### PR TITLE
CSP-427 implements before you start page

### PR DESCRIPTION
**Pull request by DevenKShah**
This PR has a lot of TODOs in place mainly because the authentication is not done yet. I will remove these comments once authentication is working. 

Also this PR includes implementation of the RegionsController. However, this was only for validation of outer api client implementation. This controller will have proper implementation and coverage as part of CSP-489.

### CODE REVIEW

File: src/SFA.DAS.ApprenticeAan.Web/Controllers/Onboarding/BeforeYouStartController.cs [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
+    }
+
 
     [HttpGet]
     public IActionResult Get()
     {
-        return View();
+        var model = new BeforeYouStartViewModel()
+        {
+            BackLink = _applicationConfiguration.ApplicationUrls.ApprenticeHomeUrl.ToString()
```
Comment (LiamWright) [inline src/SFA.DAS.ApprenticeAan.Web/Controllers/Onboarding/BeforeYouStartController.cs:28]: Isn't ApprenticeHomeUrl already a string?

**Review by LiamWright — DISMISSED** [review]
Minor comment, otherwise approved

File: src/SFA.DAS.ApprenticeAan.Web/Controllers/Onboarding/BeforeYouStartController.cs (hunk 1 above)
Comment (DevenKShah) [reply src/SFA.DAS.ApprenticeAan.Web/Controllers/Onboarding/BeforeYouStartController.cs:28]: It is a URI

**Review by LiamWright — APPROVED** [review]
Approved with code smells and coverage which will be addressed in later tickets, as discussed

**Pull request merged**